spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/MappingSpannerWriteConverter.java

```diff
@@ -32,23 +32,25 @@
 import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerMappingContext;
 import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerPersistentEntity;
 import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerPersistentProperty;
-import org.springframework.data.convert.CustomConversions;
 import org.springframework.data.mapping.PersistentPropertyAccessor;
 import org.springframework.data.mapping.PropertyHandler;
 
 /**
  * @author Balint Pato
  * @author Chengyuan Zhao
  */
-public class MappingSpannerWriteConverter extends AbstractSpannerCustomConverter
-		implements SpannerEntityWriter {
+public class MappingSpannerWriteConverter implements SpannerEntityWriter {
```

Comment:
Now that you've gone through the hard work of removing the convert functions from this, should it be renamed to `MappingSpannerWriter` ?